Source/WebCore/page/LoginStatus.h

```diff
@@ -60,6 +60,9 @@ class LoginStatus {
     WallTime loggedInTime() const { return m_loggedInTime; }
     Seconds timeToLive() const { return m_timeToLive; }
 
+    LoginStatus isolatedCopy() const & { return LoginStatus { *this }; }
```

Comment:
This doesn't look safe. It is not calling `isolatedCopy()` on any of the data members. m_domain & m_username definitely needs to be isolated copied since they are Strings (I didn't check the others in detail).